src/osyris/plot/render.py

```diff
@@ -51,15 +52,24 @@ def render(x=None, y=None, data=None, logx=False, logy=False, ax=None):
             getattr(wrappers, func)(ax, x, y, item["data"], **item["params"]))
 
         need_cbar = False
+
+        if func == "line_integral_convolution":
+            if "color" in item["params"]:
+                need_cbar = True
+
+
         if func in ["contourf", "pcolormesh"]:
             need_cbar = True
         if (func == "scatter") and ("c" in item["params"]):
             if not isinstance(item["params"]["c"], str):
                 need_cbar = True
         if need_cbar and cbar:
-            cb = plt.colorbar(mpl_objects[-1], ax=ax, cax=None)
-            cb.set_label(make_label(name=item["name"], unit=item["unit"]))
+            if func == "line_integral_convolution":
+                cb = plt.colorbar(mpl_objects[-1].images[0], ax=ax, cax=None)
+                cb.set_label(make_label(name=item["params"]["color"].name, unit=item["params"]["color"].unit))
```

Comment:
Small issue with this line of code here. It seems item["params"]["color"].unit is displaying the correct units but with a number in front of them (ex. 1.0 g / cm**3) and I'm not entirely sure how to give this a clean fix.